PagingSample/app/src/main/java/paging/android/example/com/pagingsample/CheeseItem.kt

```diff
@@ -0,0 +1,9 @@
+package paging.android.example.com.pagingsample
+
+/**
+ * Common UI model between the [Cheese] data class and separators.
+ */
+sealed class CheeseItem(val name: String) {
```

Comment:
CheeseItem doesn't seem like a clear name, since it implies it's always a cheese.

Maybe CheeseListItem, to try and convey that it's an item in the cheese list, but not necessarily a cheese?

Reply:
Done